crates/typst/src/layout/measure.rs

```diff
@@ -76,13 +76,19 @@ pub fn measure(
     height: Smart<Length>,
     /// The content whose size to measure.
     content: Content,
-    /// _Compatibility:_ This argument only exists for compatibility with
-    /// Typst 0.10 and lower and shouldn't be used anymore.
+    /// _Compatibility:_ This argument is deprecated. It only exists for
+    /// compatibility with Typst 0.10 and lower and shouldn't be used anymore.
     #[default]
     styles: Option<Styles>,
 ) -> SourceResult<Dict> {
     let styles = match &styles {
-        Some(styles) => StyleChain::new(styles),
+        Some(styles) => {
+            engine.sink.warn(warning!(
+                span, "calling `query` with a styles argument is deprecated";
```

Comment:
This is `measure`, not `query` (also in test)

Reply:
oops